### PR TITLE
README fixes: Links, etc

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,36 +129,49 @@ Available filters:
...
+
+If your question is not answered on this page:
+
+* For help with the plugin: Please use the [support forum on wordpress.org](https://wordpress.org/support/plugin/vipps-recurring-payments-gateway-for-woocommerce/) or [submit an issue](https://github.com/vippsas/vipps-recurring-woocommerce/issues) on GitHub.
+* For help with Vipps: Please see the
+  [contact us](https://github.com/vippsas/vipps-developers/blob/master/contact.md)
+  page, and also the main
+  [Vipps GitHub page](https://github.com/vippsas).
+  The
+  [Vipps eCom FAQ](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md) may also be useful.
```
Comment (Marcuzz) [inline README.md:142]: Was adding a link to the ecom API instead of the recurring API FAQ on purpose?

File: README.md [hunk 2]
```diff
@@ -68,16 +76,25 @@ The documentation for retrieving your Vipps API Keys can be found [here](https:/
...
+order in WooCommerce when you have shipped the product.
 
-The configuration for whether or not the product is virtual or physical is important to consider. 
-If a product is virtual the customer will be charged immediately but if the product is physical you will have to capture the payment manually through the order in WooCommerce when you have shipped the product.
+In most cases your products should be virtual when using subscriptions but it is
+possible to use the plugin with physical products if you need to do so.
 
-In most cases your products should be virtual when using subscriptions but it is possible to use the plugin with physical products if you need to do so.
+See the
+[Vipps eCom FAQ](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md)
```
Comment (Marcuzz) [inline README.md:91]: Was adding a link to the ecom API instead of the recurring API FAQ on purpose?

File: README.md (hunk 1 above)
Comment (cloveras) [reply README.md:150]: We need both. Fixed now, thanks.

File: README.md (hunk 2 above)
Comment (cloveras) [reply README.md:97]: We need both. Fixed now.

**Pull request merged**